symfony/plugins/orangehrmTimePlugin/Dao/TimesheetDao.php

```diff
@@ -127,7 +131,7 @@ public function hasTimesheetForStartDate(int $employeeNumber, DateTime $date): b
      * @return TimesheetActionLog[]
      */
     public function getTimesheetActionLogs(
-        int $timesheetId,
+        int                                  $timesheetId,
```

Comment:
this formatting is not matching with usual formatting. Too many unnecessary modifications there in the files cz of this. Please check your IDE configs